core/src/main/java/org/apache/carbondata/core/util/CarbonUtil.java

```diff
@@ -2814,6 +2817,22 @@ public static boolean hasAggregationDataMap(CarbonTable carbonTable) {
     return false;
   }
 
+  /**
+   * Utility function to check whether table has mv datamap or not
+   * @param carbonTable
+   * @return timeseries data map present
```

Comment:
change to `return true if MV datamap present in the specified table`

Reply:
ok , i change it

Reply:
ok， i am modifying it